packages/table/src/table.tsx

```diff
@@ -1928,7 +1927,9 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     private handleColumnsReordering = (oldIndex: number, newIndex: number, length: number) => {
         const guideIndex = Utils.reorderedIndexToGuideIndex(oldIndex, newIndex, length);
         const leftOffset = this.grid.getCumulativeWidthBefore(guideIndex);
-        this.setState({ isReordering: true, verticalGuides: [leftOffset] } as ITableState);
+        const quadrantType = guideIndex <= this.props.numFrozenColumns ? QuadrantType.TOP_LEFT : QuadrantType.TOP;
```

Comment:
Should it be strictly `<` since guideIndex is 0-index and numFrozenColumns is 1-indexed? Also, use `this.getNumFrozenColumnsClamped()`.